wildlifelicensing/apps/customer_management/static/wl/js/officer_edit_customer_details.js

```diff
@@ -0,0 +1,37 @@
+define([
+    'jQuery'
+], function ($) {
+    "use strict";
+
+    function init() {
+        var $uploadID = $('#upload_id'),
+            $formID = $('#id_form'),
+            $uploadSenior = $('#upload_senior_card'),
+            $formSenior = $('#senior_form');
+
+        $uploadID.on('click', function (e) {
+            var $fileNode = $('<input class="top-buffer" id="id" name="id" type="file" multiple>');
```

Comment:
Might be worth making this hidden (using bootstrap hidden class) so it doesn't pop in.